src/my_crypto.erl

```diff
@@ -20,16 +20,20 @@ read_file_key_full(private, FileName) ->
     Key = public_key:pem_entry_decode(RSAEntry),
     Key;
 
-    
+
 read_file_key_full(public, FileName) ->
     {ok, Bin} = file:read_file(FileName),
     [{Key, _}] = public_key:ssh_decode(Bin, public_key),
     Key.
 
+-spec potato_key() -> {crypto:rsa_public(), crypto:rsa_private()}.
+potato_key() ->
+  crypto:generate_key(rsa, {8, 65537}).
```

Comment:
you don't wanna read it from the file?
also is this compatible with my_crypto:sign function?

Reply:
idk 😱 
I couldn't figure out what params you used to generate the key files.
I didn't read from file because I want it to be a different key every time.

Reply:
I don't see why it wouldn't be compatible with my_crypto:sign function. I suppose I could write a test for it 😱

Reply:
i see, i though it wasn't random, but i suppose it is

if it is compatible, that would actually save me a lot of headache

i was reading it from files because i didn't know how to make those dynamically

Reply:
but how did you make the files :scream: ?

Reply:
same way i make all my ssh keys, ssh-gen in linux terminal 😱 😱